sampler_t.adoc

```diff
@@ -46,7 +46,7 @@ The sampler fields are described in the table below:
 
 |
 
-Specifies whether the `x`, `y` and `z` coordinates are passed in as normalized or unnormalized values.
+Specifies whether the `x`, `y` and `z` coordinates are passed in as normalized ([0,1[) or unnormalized ({0, size-1}) values.
```

Comment:
Oh, I wasn't talking about out of range coordinates. I wanted to say that since floating point coordinates work with unnormalized values, we should just say that unnormalized values are in `[0; size - 1]` range (not only integers), or maybe `[0; dimension_size - 1]`. If developer uses function with int coordinates, it's obvious she/he needs to use `intN`.